src/utilities/hw 1/getSmallestNumber/getSmallestNumber.test.js

```diff
@@ -0,0 +1,20 @@
+import { getSmallestNumber } from './getSmallestNumber';
+
+describe('The getSmallestNumber function', () => {
+  describe('when correct array is passed as argument', () => {
+    const array = [5, 7, -9, 0, 65, 134];
```

Comment:
```suggestion
    const smallestNumber = -9;
    const array = [5, 7, smallestNumber, 0, 65, 134];
```
create a variable holding -9 and use it to create the array and in the assertion

Reply:
done